### PR TITLE
Image messages

### PR DESCRIPTION
**Pull request by SibaIssa**

### CODE REVIEW

File: Runtime/RosMsgs.cs [hunk 1]
```diff
@@ -388,6 +388,182 @@ public class NavSatFix : IRosMsg, IStamped
...
+        /// Acceptable values:
+        /// jpeg, png
+        /// </summary>
+        public string format;
+
+        /// <summary>
+        /// Compressed image buffer
+        /// </summary>
+        public byte[] data;
+    }
```
Comment (Sh1co) [inline Runtime/RosMsgs.cs:527]: Compressed Image msg will be added in another PR,  I think we should remove it to avoid conflict

**Review by Sh1co — CHANGES_REQUESTED** [review]

**Review by xovobobo — APPROVED** [review]

**Review by Sh1co — APPROVED** [review]
